migration/migration/006-matcher-operators.sql

```diff
@@ -128,7 +131,33 @@ AS $func$
     FROM _prom_catalog.label l
     WHERE l.key = key_to_match and l.value ~ pat
 $func$
-LANGUAGE SQL STABLE PARALLEL SAFE;
+LANGUAGE SQL STABLE PARALLEL SAFE
+SUPPORT _prom_ext.rewrite_fn_call_to_subquery;
+
+-- TODO (james): should these operators be in the _prom_catalog schema, or a "public" schema?
```

Comment:
schemas starting with underscores are considered internal. Since these operators are meant to be used by end-users, I think they should go in one of our metrics-specific public schemas.

Reply:
this has been removed, see Mat's comment